src/thunderbots/software/backend_output/main.cpp

```diff
@@ -60,7 +79,7 @@ int main(int argc, char** argv)
         // The callbacks will populate the primitives vector
         ros::spinOnce();
 
-        backend.sendPrimitives(primitives);
+        backend.sendPrimitives(primitives, friendly_team);
```

Comment:
(2/2) So here, update the grSim backend with `friendly_team` in a separate function.

Reply:
Sorry, not quite sure what you mean. Put line 52 in a separate function?

Reply:
Remove the `friendly_team` dependency from `sendPrimitives`, and have a separate function in backend that updates `friendly_team`.
Something like:
```
backend.updateTeam(friendly_team); 
backend.sendPrimitives(primitives);
```

Reply:
See above. updateBackendTeam() helper function created